express_import/main.ipynb

```diff
@@ -0,0 +1,197 @@
+    "    print(url)\n",
+    "    response = requests.get(url, headers=headers)\n",
+    "    if response.status_code == 200:\n",
+    "        return response.json()['payload']\n",
+    "    else:\n",
+    "        raise Exception(f\"Failed to list tasks: {response.json()}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "18425218-50ce-41a7-a6a2-becb09ce4177",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Function to run the express import into tasks\n",
+    "def add_import(key, token, project_id, tasks, uri, region):\n",
+    "    headers = {\"Authorization\": f\"Bearer {token}\", \"X-API-KEY\": key}\n",
+    "    payload = {\n",
+    "          \"project_id\": project_id,\n",
+    "          \"source_type\": \"s3\",\n",
+    "          \"source_region\": region,\n",
+    "          \"uri\": uri,\n",
+    "          \"target_task_ids\": tasks\n",
+    "    }\n",
+    "    response = requests.post(imports_url, headers=headers, json=payload)\n",
+    "    if response.status_code == 200:\n",
+    "        print(\"Import added successfully:\", response.json())\n",
+    "    else:\n",
+    "        raise Exception(f\"Failed to add import: {response.json()}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "53d10c51-bc62-4d89-8455-b0d2c6e8cf77",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "An error occurred: Authentication failed: {'message': 'Unauthorized'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Main script to run the process\n",
+    "if __name__ == \"__main__\":    \n",
+    "    \n",
+    "    try:\n",
+    "        # Step 1: Authenticate and get token\n",
+    "        token = get_auth_token(email, api_password, api_key)\n",
+    "        print(f\"Authentication successful with token {token}\")\n",
+    "\n",
+    "        # Step 2: Get project ID from the user\n",
+    "        project_id = input(\"Enter the project ID: \")\n",
+    "        \n",
+    "        # Step 3: List all tasks in the project\n",
+    "        all_tasks = list_tasks(token, api_key, project_id)\n",
+    "        print(f\"Found {len(all_tasks)} tasks in the project {project_id}\")\n",
+    "        \n",
+    "        for task in all_tasks:\n",
+    "            print(f\"Task ID: {task['task_id']}, Name: {task['task_name']}\")\n",
+    "\n",
+    "        # Step 4: input tasks to add\n",
+    "        task_input = input(\"Enter the tasks IDs you want to import to separated by commas: \")\n",
+    "        tasks = [\n",
+    "            int(task.strip()) for task in task_input.split(\",\") if task.strip()\n",
+    "        ]\n",
+    "        \n",
+    "        # Step 5: Add an import to the tasks\n",
+    "        if tasks:\n",
+    "            add_import(api_key, token, project_id, tasks, s3_uri, s3_region)\n",
+    "        else:\n",
+    "            print(\"No tasks to add import to\")\n",
```

Comment:
We should still make the import request, but just skip the `target_task_ids` input.